tools/c7n_terraform/readme.md

```diff
@@ -0,0 +1,72 @@
+
+# Cloud Custodian Terraform Provider
+
+Custodian's terraform provider enables writing and evaluating
+custodian policies against Terraform IaaC modules.
+
+
+tldr: we want to enable writing custodian policies against IaaC assets (terraform, cfn, etc) directly in devops/ci pipelines.
+
+# Purpose
+
+The primary purpose of this is to integrate with ci/cd pipelines to
+evaluate compliance and governance early in the deployment
+lifecycle. Custodian cloud providers provide for realtime detection
+and remediation as a detective control against infrastructure already
+deployed in the environment regardless of how it was provisioned. As
+an initial target, the terraform provider is designed to complement
+that with preventive enforcement earlier in the
+lifecycle. ie. enabling a shift-left to policy enforcement.
+
+
+# Pipeline CLI
+
+In looking at expanding out to shift-left pipeline use cases, one
```

Comment:
It feels to me that supporting CI/CD pipelines would be better done with a custom sub-command to custodian.